user/src/com/google/gwt/user/client/impl/DOMImpl.java

```diff
@@ -102,7 +102,7 @@ public native int eventGetTypeInt(String eventType) /*-{
     case "scroll": return 0x04000;
     case "error": return 0x10000;
     case "mousewheel": return 0x20000;
```

Comment:
To confirm, the `mousewheel` case is left here in case someone creates their own mouse event type using `MouseEvents.MOUSEWHEEL`?

Reply:
Yes.